plugins/defaultplugins/srplugin/vppcalls/srv6.go

```diff
@@ -0,0 +1,510 @@
+// Copyright (c) 2018 Bell Canada, Pantheon Technologies and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vppcalls
+
+import (
+	"fmt"
+	"net"
+	"strings"
+	"time"
+
+	"github.com/ligato/cn-infra/logging"
+	"github.com/ligato/cn-infra/logging/measure"
+	"github.com/ligato/vpp-agent/plugins/defaultplugins/common/bin_api/sr"
+	"github.com/ligato/vpp-agent/plugins/defaultplugins/common/model/srv6"
+	"github.com/ligato/vpp-agent/plugins/defaultplugins/ifplugin/ifaceidx"
+)
+
+// Constants for behavior function hardcoded into VPP (there can be also custom behavior functions implemented as VPP plugins)
+// Constants are taken from VPP's vnet/srv6/sr.h (names are modified to Golang from original C form in VPP code)
+const (
+	BehaviorEnd    uint8 = iota + 1 //Behavior of simple endpoint
```

Comment:
Comments should start with space.

Reply:
fixed in df0c05a74d21e7bab2f751f0afaa27a60d7f3e7c